packages/babel-helper-replace-supers/src/index.ts

```diff
@@ -524,7 +524,7 @@ export default class ReplaceSupers {
     this.constantSuper = process.env.BABEL_8_BREAKING
       ? opts.constantSuper
       : // Fallback to isLoose for backward compatibility
-        opts.constantSuper ?? (opts as any).isLoose;
```

Comment:
I'm curious about what caused this, since this PR does not update prettier.

Reply:
Ah, my bad.
I updated prettier too. :)

Reply:
It's fine, just pointing it out!